packages/ai/src/ai/modules/task/task_server.py

```
@@ -581,43 +608,19 @@ def release_port(self, port: int) -> None:
-            listen_type (EVENT_TYPE): Category of event being broadcast
-                                    (PASSIVE, ACTIVE, DEBUG, etc.)
-            id (str): Short identifier for the specific task instance
-            token (str): Unique task identifier for event source
-            event (Dict[str, Any]): Event payload containing event details and data
-
-        Event Distribution Logic:
-        - Iterates through all active connections
-        - Each connection filters events based on subscription preferences
-        - Individual broadcast failures don't affect other recipients
-        - Events are only sent to connections with matching access rights
+    async def broadcast_server_event(self, type: EVENT_TYPE, event: Dict[str, Any]) -> None:
```

Ruff (0.15.7)

[error] Function argument `type` is shadowing a Python builtin

(A002)